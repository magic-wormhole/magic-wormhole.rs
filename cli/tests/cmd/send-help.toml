bin.name = "wormhole-rs"
args = "send --help"
