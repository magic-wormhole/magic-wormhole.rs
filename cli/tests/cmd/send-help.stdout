Send a file or a folder

Usage: wormhole-rs[EXE] send [OPTIONS] <FILENAME|DIRNAME>...

Arguments:
  <FILENAME|DIRNAME>[..]

Options:
...